src/components/dynamic-inspector-controls/dynamic-inspector-controls.js

```diff
@@ -207,32 +224,20 @@ const DynamicInspectorControls = ({ attributes, setAttributes, config, setIsDirt
           })}
         </PanelBody>
       )}
-      {availableTaxsAndTerms && availableTaxsAndTerms.length ? (
+      {availableTaxonomiesWithTerms && availableTaxonomiesWithTerms.length ? (
```

Comment:
En detalj och en smaksak men det går ju att skriva `availableTaxonomiesWithTerms?.length` för att spara in på lite kod :) @perarnborg

Reply:
@abeijnoff Håller med! Vi följde mönstret sedan tidigare. Men det är ju inte fel att ta chansen att göra en fil snyggare en den var innan när man lämnar tillbaka den. 👍

Reply:
Lite mindre js-grejjer fixade nu, bland annat denna! Bra span Axel!